misc/ns-configurator/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <numeric>
+
+#include "AbstractConfig.h"
+#include "NsCmdLineParameters.h"
+
+using namespace std;
+using namespace nanoservices;
+
+int main(int argc, char** argv) {
+	map<char, NsCmdLineParameters::opt> prms = {
+		{'p' , {"path", 'p', "PATH", 0, "Path to config node", 2}},
+		{'a' , {"data", 'a', "DATA", 0, "Data for create or update operation", 2}},
+		{'c' , {"create", 'c', 0, 0, "create config node at PATH with DATA", 1}},
+		{'r' , {"read", 'r', 0, 0, "read config node data from PATH", 1}},
+		{'u' , {"update", 'u', 0, 0, "update config node at PATH as DATA", 1}},
+		{'d' , {"delete", 'd', 0, 0, "delete config node at PATH", 1}},
```

Comment:
Лучше оставить только реально поддерживаемые на данный момент, остальные закомментировать
```suggestion
//TODO		{'p' , {"path", 'p', "PATH", 0, "Path to config node", 2}},
//TODO		{'a' , {"data", 'a', "DATA", 0, "Data for create or update operation", 2}},
		{'c' , {"create", 'c', 0, 0, "create config node at PATH with DATA", 1}},
		{'r' , {"read", 'r', 0, 0, "read config node data from PATH", 1}},
//TODO		{'u' , {"update", 'u', 0, 0, "update config node at PATH as DATA", 1}},
		{'d' , {"delete", 'd', 0, 0, "delete config node at PATH", 1}},
```

Reply:
Что значит реально поддерживаемые?
Всё что надо это написать 3 строки кода про обновление

Reply:
Добавил выполнение update, завернул всё в try